examples/jl-lab-extension-slate-steps.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
<p>This file seems a copy of slate-demo-explicit, except with <code>make_lab_client</code> instead of <code>make_client</code>  You might wanna rename the filename's last part from <code>-slate-steps</code> to <code>-demo-explicit2</code>   like  the case with other file (it took me a while to follow)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Caltech-IPAC/firefly_client/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>